### PR TITLE
fix: fix content-claims package not being importable by ts project by having package.json point to src alongside types

### PR DESCRIPTION
**Pull request by gobengo**
Motivation:
* https://github.com/web3-storage/content-claims/issues/40

Notes
* when I do `npm pack` of the packages/core in here and then `npm install file:<path-to-it>` in hoverboard, it resolves the issue

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -2,13 +2,13 @@
   "name": "@web3-storage/content-claims",
   "version": "4.0.0",
   "description": "Implementation of the Content Claims Protocol.",
-  "main": "src/index.js",
+  "main": "./types/index.js",
```
Comment (travis) [inline packages/core/package.json:5]: in this case should we rename types to dist? feels weird having the source in a folder called types...

**Review by travis — APPROVED** [review]
I'm a hesitant +1 - have a fairly strong preference for renaming the build directory, but can live with this if that's a hassle - probably worth creating an issue for it if not addressed here

File: packages/core/package.json (hunk 1 above)
Comment (gobengo) [reply packages/core/package.json:5]: i'm open to it but i didn't want to change it from what it was and wanted to get ideas. I agree types is gross. dist is conventional and fine so it's a good option. I was thinking what if we start using `lib`.... but that's when I decided not to do it in this PR :P

Comment (travis) [reply packages/core/package.json:5]: I like `dist` fwiw!

**Pull request merged**